charts/tvheadend/values.yaml

```diff
@@ -0,0 +1,30 @@
+# Default values for tvheadend.
+
+image:
+  repository: linuxserver/tvheadend
+  pullPolicy: IfNotPresent
+  tag: release-4.2
```

Comment:
Actually looking at LSIO image tags for this image, I would roll with `tag: latest` here.

Reply:
Done